.circleci/config.yml

```diff
@@ -198,7 +198,7 @@ jobs:
           test-targets: notPackage org.wordpress.android.ui.screenshots
           device: model=Pixel2,version=28,locale=en,orientation=portrait
           project: api-project-108380595987
-          timeout: 10m
+          timeout: 20m
```

Comment:
One of the tests was being interrupted because of the `10m` timeout. As we have more UI tests coming, I've set it to `20m` so we have enough room.

Reply:
You're right. This is the limit for the whole Connected tests step and it fails if it's still running after this time. I had to add more time because one of the two tests being added was already being interrupted. 

I went from `10m` to `20m` because we've been adding UI Tests and `15m`, for example, felt too tight as a single UI Test could take ~3m to run with `setUp` and `tearDown`.

Here's an example of what a timeout looks like [in CircleCI](https://app.circleci.com/pipelines/github/wordpress-mobile/WordPress-Android/28467/workflows/147f00a8-df61-4859-91d7-9358b5d80f82/jobs/118200/steps) and [in Firebase](https://console.firebase.google.com/u/0/project/api-project-108380595987/testlab/histories/bh.e0c3a59bd9ed670/matrices/5854000507092923468/executions/bs.c129e943dc1c19cb).